server.py

```diff
@@ -43,7 +74,7 @@ def bad_request(error):
 
 @app.errorhandler(404)
 def not_found(error):
-    """ Handles Pets that cannot be found """
+    """ Handles product information that cannot be found """
     return jsonify(status=404, error='Not Found', message=error.message), 404
```

Comment:
Since we are cleaning the code here, maybe we can replace the status code with constant variable too (e.g. 404 -> status.HTTP_404_NOT_FOUND etc). The same goes with all other error handler code.

Reply:
Addressed